### DISCUSSION TITLE
so-zeek continually failing to start

### DISCUSSION THREAD

**Discussion by Kyle-44** [Security-Onion-Solutions/securityonion#13445] [Unsupported Versions]
Hello,

2.4.90 Installed (Was updated from 2.4.60 thinking this would be resolved with an upgrade)
Distributed architecture with 1 forwarder, 1 search, and 1 manager

I am having an issue with Zeek failing to start scripts. 
When running "docker logs so-zeek" I get this output:

docker logs so-zeek
checking configurations ...
logger scripts failed.
error in /opt/zeek/share/zeek/policy/cve-2020-0601/./cve-2020-0601.zeek, line 1: syntax error, at or near "module" or end of file /opt/zeek/share/zeek/policy/cve-2020-0601/__load__.zeek

manager scripts failed.
error in /opt/zeek/share/zeek/policy/cve-2020-0601/./cve-2020-0601.zeek, line 1: syntax error, at or near "module" or end of file /opt/zeek/share/zeek/policy/cve-2020-0601/__load__.zeek

proxy scripts failed.
error in /opt/zeek/share/zeek/policy/cve-2020-0601/./cve-2020-0601.zeek, line 1: syntax error, at or near "module" or end of file /opt/zeek/share/zeek/policy/cve-2020-0601/__load__.zeek

worker-1-1 scripts failed.
error in /opt/zeek/share/zeek/policy/cve-2020-0601/./cve-2020-0601.zeek, line 1: syntax error, at or near "module" or end of file /opt/zeek/share/zeek/policy/cve-2020-0601/__load__.zeek

/usr/local/sbin/zeek.sh: line 8: kill: (31) - No such process
checking configurations ...
logger scripts failed.
error in /opt/zeek/share/zeek/policy/cve-2020-0601/./cve-2020-0601.zeek, line 1: syntax error, at or near "module" or end of file /opt/zeek/share/zeek/policy/cve-2020-0601/__load__.zeek

manager scripts failed.
error in /opt/zeek/share/zeek/policy/cve-2020-0601/./cve-2020-0601.zeek, line 1: syntax error, at or near "module" or end of file /opt/zeek/share/zeek/policy/cve-2020-0601/__load__.zeek

proxy scripts failed.
error in /opt/zeek/share/zeek/policy/cve-2020-0601/./cve-2020-0601.zeek, line 1: syntax error, at or near "module" or end of file /opt/zeek/share/zeek/policy/cve-2020-0601/__load__.zeek

worker-1-1 scripts failed.
error in /opt/zeek/share/zeek/policy/cve-2020-0601/./cve-2020-0601.zeek, line 1: syntax error, at or near "module" or end of file /opt/zeek/share/zeek/policy/cve-2020-0601/__load__.zeek

/usr/local/sbin/zeek.sh: line 8: kill: (31) - No such process
checking configurations ...
logger scripts failed.
error in /opt/zeek/share/zeek/policy/cve-2020-0601/./cve-2020-0601.zeek, line 1: syntax error, at or near "module" or end of file /opt/zeek/share/zeek/policy/cve-2020-0601/__load__.zeek

manager scripts failed.
error in /opt/zeek/share/zeek/policy/cve-2020-0601/./cve-2020-0601.zeek, line 1: syntax error, at or near "module" or end of file /opt/zeek/share/zeek/policy/cve-2020-0601/__load__.zeek

proxy scripts failed.
error in /opt/zeek/share/zeek/policy/cve-2020-0601/./cve-2020-0601.zeek, line 1: syntax error, at or near "module" or end of file /opt/zeek/share/zeek/policy/cve-2020-0601/__load__.zeek

worker-1-1 scripts failed.
error in /opt/zeek/share/zeek/policy/cve-2020-0601/./cve-2020-0601.zeek, line 1: syntax error, at or near "module" or end of file /opt/zeek/share/zeek/policy/cve-2020-0601/__load__.zeek

/usr/local/sbin/zeek.sh: line 8: kill: (30) - No such process
checking configurations ...
logger scripts failed.
error in /opt/zeek/share/zeek/policy/cve-2020-0601/./cve-2020-0601.zeek, line 1: syntax error, at or near "module" or end of file /opt/zeek/share/zeek/policy/cve-2020-0601/__load__.zeek

manager scripts failed.
error in /opt/zeek/share/zeek/policy/cve-2020-0601/./cve-2020-0601.zeek, line 1: syntax error, at or near "module" or end of file /opt/zeek/share/zeek/policy/cve-2020-0601/__load__.zeek

proxy scripts failed.
error in /opt/zeek/share/zeek/policy/cve-2020-0601/./cve-2020-0601.zeek, line 1: syntax error, at or near "module" or end of file /opt/zeek/share/zeek/policy/cve-2020-0601/__load__.zeek

worker-1-1 scripts failed.
error in /opt/zeek/share/zeek/policy/cve-2020-0601/./cve-2020-0601.zeek, line 1: syntax error, at or near "module" or end of file /opt/zeek/share/zeek/policy/cve-2020-0601/__load__.zeek

I attempted to find similar errors in the discussion and tried "sudo salt-call state.highstate" with no errors.

Ran "sudo grep community /opt/so/saltstack/default/pillar/zeek/init.sls" & "sudo grep -r communityid /opt/so" and got no response. 

I am fairly new with troubleshooting Sec Onion issues, please advise.

**Comment by cm-ops** [comment]
If you `cat /opt/so/conf/zeek/policy/cve-2020-0601/cve-2020-0601.zeek` what does it show on line 1?

**Comment by Kyle-44** [comment]
module CVE_2020_0601;

export {
        ## set to yes, to log suspicious certificates.
        option log_certs = F;

        ## The logging stream identifier.
        redef enum Log::ID += { LOG };

        ## The record type which contains column fields of the certificate log.
        type Info: record {
                ## Timestamp when this record is written.
                ts:          time   &log;
                ## File-id of the cerfificate
                fuid:        string &log;
                ## Certificate encoded as base64
                certificate: string &log;
        };

        redef enum Notice::Type += {
                ## An ECC certificate with an unknown curve was encountered
                Unknown_X509_Curve
        };
}

event zeek_init()
        {
        Log::create_stream(CVE_2020_0601::LOG, [$columns=Info, $path="cve-2020-0601-certs"]);
        }

event x509_certificate(f: fa_file, cert_ref: opaque of x509, cert: X509::Certificate)
        {
        if ( cert?$key_alg && cert$key_alg == "id-ecPublicKey" && ! cert?$curve )
                {
                NOTICE([$note=Unknown_X509_Curve, $f=f, $msg="ECC certificate with unknown curve; potential CVE-2020-0601 exploit attempt"]);

                if ( log_certs )
                        Log::write(CVE_2020_0601::LOG, Info($ts=network_time(), $fuid=f$id, $certificate=encode_base64(x509_get_certificate_string(cert_ref, F))));
                }
        }


Thank you for your assistance!

**Reply by cm-ops** [reply]
Okay, looks like that didn't paste so well.

Can you let me know what this looks like? `Administration > Configuration > zeek > config > local > load`
